### ISSUE TITLE
Unable To Add A Role To A User

### ISSUE THREAD

**Issue by mawesome4ever** [hydrabolt/discord.js#1312]
Hello, when trying to add a role to a user who joins i get an "Cannot read property 'id' of undefined".
This is my code:
`
bot.on('guildMemberAdd',(member) => {
	member.addRole(296233315059630091).catch(console.error);
});
`

and this is the output:

`/Users/myself/Desktop/verificationBot/discord.io-master/node_modules/discord.js/src/client/rest/RE
STMethods.js:323
    return this.rest.makeRequest('put', Constants.Endpoints.guildMemberRole(member.guild.id, member.
id, role.id), true)

        ^

TypeError: Cannot read property 'id' of undefined
    at RESTMethods.addMemberRole (/Users/myself/Desktop/verificationBot/discord.io-master/node_mod
ules/discord.js/src/client/rest/RESTMethods.js:323:109)
    at CommandoClient.bot.on (/Users/myself/Desktop/verificationBot/discord.io-master/lib/index.js
:16:9)
    at emitOne (events.js:96:13)
    at CommandoClient.emit (events.js:191:7)
    at Guild._addMember (/Users/myself/Desktop/verificationBot/discord.io-master/node_modules/disc
ord.js/src/structures/Guild.js:783:19)
    at GuildMemberAddHandler.handle (/Users/myself/Desktop/verificationBot/discord.io-master/node_
modules/discord.js/src/client/websocket/packets/handlers/GuildMemberAdd.js:12:13)
    at WebSocketClient.internalOnMessage (/Users/myself/Desktop/verificationBot/discord.io-master/
node_modules/uws/uws.js:89:17)
    at native.client.group.onMessage (/Users/myself/Desktop/verificationBot/discord.io-master/node
_modules/uws/uws.js:48:15)`

I'm new to javascript, if you could explain to me the error and a possible way to fix it in laymans terms i will greatly appreciate it. Thank you in advance!

**Comment by mawesome4ever** [comment]
Sorry, i was testing to see if the error would go away. The error still appears with quotes.

**Comment by Anonymous0perator** [comment]
Well, This is easy

**Comment by dragonfire535** [comment]
Can you please read dates before posting on issues from months ago?
